app/views/layouts/_attention.html.erb

```diff
@@ -1,6 +1,6 @@
 <% errors = collect_errors %>
 
-<div class="errors">
+<div class="errors" aria-live="polite">
   <%= alert_tag(errors[:alerts])   %>
   <%= notice_tag(errors[:notices]) %>
 </div>
```

Comment:
Hey @RoyEJohnson would `alert` be a better role for these?  I'm not sure but that seems to fit the purpose of the errors div

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Reply:
Reading about it, I think so. I'll put that in.

Reply:
Yeah, I'd never used anything other than `polite` before but it seemed appropriate here 😁